po/de.po

```diff
@@ -6365,33 +6369,39 @@ msgstr ""
 
 #: ../src/preferences.cpp:492
 msgid "VapourSynth"
-msgstr ""
+msgstr "VapourSynth"
 
 #: ../src/preferences.cpp:497
 msgid "Log Level"
-msgstr ""
+msgstr "Log-Level"
 
 #: ../src/preferences.cpp:499
 msgid "Default Video Script"
-msgstr ""
+msgstr "Standard-Videoskript"
 
 #: ../src/preferences.cpp:501
 msgid ""
 "This script will be executed to load video files that aren't\n"
 "VapourSynth scripts (i.e. end in .py or .vpy).\n"
 "The filename variable stores the path to the file."
 msgstr ""
+"Dieses Skript wird ausgeführt um Videodateien zu laden,\n"
+"die nicht bereits VapourSynth-Skripte sind (.py- oder .vpy-Endung).\n"
+"Die Variable speichert den Dateipfad."
 
```

Comment:
In the original message, `filename` is the name of the variable in the Python script. So the third line should be something like `Die Variable filename speichert den Dateipfad.`, possibly also with quotes around it.

Same for the audio script message at `preferences.cpp:510`